repos/system_upgrade/el7toel8/actors/localreposinhibit/actor.py

```diff
@@ -0,0 +1,72 @@
+from leapp import reporting
+from leapp.actors import Actor
+from leapp.models import TMPTargetRepositoriesFacts, UsedTargetRepositories
+from leapp.reporting import Report
+from leapp.tags import IPUWorkflowTag, TargetTransactionChecksPhaseTag
+from leapp.utils.deprecation import suppress_deprecation
+
+
+@suppress_deprecation(TMPTargetRepositoriesFacts)
+class LocalReposInhibit(Actor):
+    """Inhibits the upgrade if local repositories were found."""
+
+    name = "local_repos_inhibit"
+    consumes = (
+        UsedTargetRepositories,
+        TMPTargetRepositoriesFacts,
+    )
+    produces = (Report,)
+    tags = (IPUWorkflowTag, TargetTransactionChecksPhaseTag)
+
+    def file_baseurl_in_use(self):
+        """Check if any of target repos is local.
+
+        UsedTargetRepositories doesn't contain baseurl attribute. So gathering
+        them from model TMPTargetRepositoriesFacts.
+        """
+        used_target_repos = next(self.consume(UsedTargetRepositories)).repos
+        target_repos = next(self.consume(TMPTargetRepositoriesFacts)).repositories
+        target_repo_id_to_url_map = {
+            repo.repoid: repo.baseurl
+            for repofile in target_repos
+            for repo in repofile.data
+            if repo.baseurl
+        }
+        return any(
+            target_repo_id_to_url_map[repo.repoid].startswith("file:")
+            for repo in used_target_repos
+        )
+
+    def process(self):
+        if self.file_baseurl_in_use():
+            warn_msg = (
+                "Local repository found (baseurl starts with file:///). "
+                "Currently leapp is not supporting this option."
+            )
+            self.log.warning(warn_msg)
+            reporting.create_report(
+                [
+                    reporting.Title("Local repository identified"),
+                    reporting.Summary(warn_msg),
+                    reporting.Severity(reporting.Severity.HIGH),
+                    reporting.Tags([reporting.Tags.REPOSITORY]),
+                    reporting.Flags([reporting.Flags.INHIBITOR]),
+                    reporting.Remediation(
+                        hint=(
+                            "By using Apache HTTP Server you can expose "
+                            "your local repository via http. See the linked "
+                            "article for details. "
+                        )
+                    ),
+                    reporting.ExternalLink(
```

Comment:
The external link does not show up in the txt report. @shaded-enmity, @pirat89, is that expected?
```
Risk Factor: high (inhibitor)
Title: Local repository identified
Summary: Local repository found (baseurl starts with file:///). Currently leapp is not supporting this option.
Remediation: [hint] By using Apache HTTP Server you can expose your local repository via http. See the linked article for details. 
----------------------------------------
```

Reply:
I see at https://github.com/oamg/leapp/blob/af8730d5076659bcdc4ab0c2b09b6b408e95dcd3/leapp/utils/report.py#L91 that this is looks expected. JSON report should be complete, but txt one omitting some fields

Reply:
@bocekm It's expected now. I already raised that topic in the past, to print the external link in .txt report as well, but not sure why it is not there till this day. I guess that it's just because we missed that. But not sure really.

Reply:
ref https://github.com/oamg/leapp-repository/issues/559

Reply:
Any further discussion, let's keep in #559, because it is out of the scope of this MR